app/Types/Cause.php

```diff
@@ -46,20 +52,25 @@ public static function labels()
             self::EDUCATION => 'Education (access, affordability) ',
```

Comment:
Should `DISCRIMINATION` be added above `EDUCATION`?

Reply:
Oh jeez, it sure should!